addons/payment/tests/test_payment_token.py

```diff
@@ -39,3 +40,33 @@ def test_display_name_is_not_padded(self):
         """ Test that the display name is not padded when `should_pad` is `False`. """
         token = self._create_token()
         self.assertEqual(token._build_display_name(should_pad=False), '1234')
+
+    def test_public_user_has_access_to_own_tokens(self):
+        token = self._create_token(partner_id=self.public_user.partner_id.id)
+        token.read()
```

Comment:
I salute the effort, but I think this test is too "basic" to be kept. This is the kind of regression that we'd immediately spot if it happened. We should make a selection of the tests that we deem important and remove the others to mitigate the technical debt and testing time.

Reply:
understood :)